Runner/suites/Multimedia/OpenCV/run.sh

```diff
@@ -0,0 +1,42 @@
+# Copyright (c) Qualcomm Technologies, Inc. and/or its subsidiaries.
+# SPDX-License-Identifier: BSD-3-Clause-Clear
+
+#!/bin/sh
+# Import test suite definitions
+. $(pwd)/init_env
+TESTNAME="Opencv_core"
+
+# Import test functions
+. $TOOLS/functestlib.sh
+
+test_path=$(find_test_case_by_name "$TESTNAME")
+log_info "-----------------------------------------------------------------------------------------"
+log_info "-------------------Starting $TESTNAME Testcase----------------------------"
+
+
+log_info "Checking if dependency binary is available"
+check_dependencies opencv_test_core
+
+# Navigate to the directory where the fastrpc_test application is located
+cd /usr/bin/
+chmod 777 opencv_test_core
```

Comment:
777 permissions are excessive. After copying the file should already have 755 permissions.